src/js/plugins/plugin.resume.js

```diff
@@ -0,0 +1,88 @@
+/* global BookReader */
+
+/**
+ * Plugin to remember the current page number in a cookie
+ */
+jQuery.extend(BookReader.defaultOptions, {
+  enablePageResume: true,
+  /**
+   * @const null|string eg '/', '/details/id'
+   */
+  resumeCookiePath: null,
+});
+
+/**
+ * @override BookReader.init
+ */
+BookReader.prototype.init = (function(super_) {
+  return function() {
+    super_.call(this);
+    if (this.options.enablePageResume) {
+      this.bind(BookReader.eventNames.fragmentChange, function() {
+        const params = this.paramsFromCurrent();
+        this.updateResumeValue(params.index);
+      }.bind(this));
```

Comment:
No longer need `.bind`!

```suggestion
      this.bind(BookReader.eventNames.fragmentChange, () => {
        const params = this.paramsFromCurrent();
        this.updateResumeValue(params.index);
      });
```